celery/apps/beat.py

```diff
@@ -142,15 +157,15 @@ def startup_info(self, service):
             max_interval=scheduler.max_interval,
         )
 
-    def set_process_title(self):
+    def set_process_title(self) -> None:
         arg_start = 'manage' in sys.argv[0] and 2 or 1
         platforms.set_process_title(
             'celery beat', info=' '.join(sys.argv[arg_start:]),
         )
 
-    def install_sync_handler(self, service):
+    def install_sync_handler(self, service: Service) -> None:
```

Comment:
Pre-commit was replacing quoted type hints. Is this intended? I think it would be useful to allow them, so that I could conditionally import types with `if TYPE_CHECKING`.